app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        print("some informative message")
+        return
+
+    source_file, destination_file = parts[1], parts[2]
+
+    if source_file == destination_file:
+        print("some informative message")
+        return
```

Comment:
Mentor ment, that you should print some information about what's going on.  Use some descriptive prints in this case,please

Reply:
Ohh, okey